kadi_apps/settings/devel.py

```diff
@@ -3,7 +3,7 @@
 DEBUG = True
 TESTING = False
 
-KADI_APPS_CONFIG_DIR = '/Users/javierg/SAO/flask_apps/api-data/web-kadi-test/config'
+KADI_APPS_CONFIG_DIR = '/Users/aldcroft/git/kadi-apps/config'
```

Comment:
@javierggt - I'm not sure what is going on here but we need a way to generalize this for non-Javiers.

Reply:
Right. But so far the only "devel" has been me (note that the flight config should point to a file owned by kadi). In general it is not clear to me what the process should be for kadi-test.

I think devel configurations could use some path in /ska/data with kadi-web access, or something like that.

Reply:
Updated in https://github.com/sot/kadi-apps/pull/18/files